src/petab_gui/controllers/mother_controller.py

```diff
@@ -369,6 +374,20 @@ def setup_actions(self):
             ))
         )
 
+        # Undo / Redo
+        actions["undo"] = QAction(
+            qta.icon("mdi6.undo"),
+            "&Undo", self.view
+        )
+        actions["undo"].setShortcut("Ctrl+Z")
+        actions["undo"].triggered.connect(self.undo_stack.undo)
+        actions["redo"] = QAction(
+            qta.icon("mdi6.redo"),
+            "&Redo", self.view
+        )
+        # set shortcut as ctrl + z + Arrow up
+        actions["redo"].setShortcut("Ctrl+Shift+Z")
```

Comment:
better to use the default shortcuts, since on windows the default shortcut for redo is ctrl+y. so rather than defining them yourself, it would be best to take one of the default qkeysequences: 

<https://doc.qt.io/qt-6/qkeysequence.html#StandardKey-enum>

Reply:
Thanks, adapted all shortcuts where possible